public/Assets/Styles/styles.css

```diff
@@ -1,9 +1,101 @@
+<<<<<<< HEAD
+.related {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 auto;
+  padding: 5px;
+  width: 75%;
+  height: 10%;
+}
+.modal-body {
+  margin: 0 15%;
+  position: fixed;
+  z-index: 1;
+  width: 40%;
+  height: 25%;
+  border: 1px solid;
+  border-radius: 5px;
+  background-color: darkgray;
+}
+.table {
+  overflow-y: auto;
+  overflow-x: hidden;
+}
+.table #close {
+  margin: 0.5% 98%;
+}
+.table #close:hover {
+  color: rgba(207, 8, 8, 0.842);
+}
+.table table {
+  margin: 2% auto;
+  width: 80%;
+  height: 80%;
+}
+.table table, thead, td {
+  border: 2px solid #ddd;
+  border-collapse: collapse;
+  text-align: center;
+}
+.table thead, td {
+  padding: 4px;
+}
+.arrow-left {
+  display: inline;
+  padding-top: 25px;
+  padding-bottom: 25px;
+}
+.card-container {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  padding-top: 1%;
+  padding-bottom: 1%;
+  width: 80%;
+  border: 1px solid;
+  background-color: aquamarine;
+}
+.product-card {
+  display: inline-block;
+  margin: 0.75% 4%;
+  padding: 5px;
+  width: 15%;
+  border: 1px solid;
+  border-radius: 5px;
+  background-color: silver;
+}
+.product-card .action {
+  margin: 0 90%;
+}
+.product-card img {
+  margin: 0 6%;
+  width: 150px;
+  height: 112.5px;
+  border: 1px solid;
+  overflow: hidden;
+}
+.product-card p {
+  text-align: center;
+}
+
+.action:hover {
+  color: cyan;
+}
+.arrow-right {
+  display: inline;
+  padding-top: 25px;
+  padding-bottom: 25px;
+}
+/* --------- Stars-Display ----------- */
+=======
 @import 'carousel.css';
 @import 'modal.css';
 @import 'related-products.css';
 
+>>>>>>> f47b48c19a974d4f3868f164a4a8b34c6eeb0b09
```

Comment:
can you just make sure this isn't showing as a merge conflict? It just looks weird.